working-groups.md

```diff
@@ -0,0 +1,65 @@
+# Working Groups 
+
+### [Business Planning][biz-wg]
+
+WG for performing feasibility study, and business planning to identify start up costs, fundraising mechanisms, and risks.
+
+📅 Weekly calls  
+🌱 Cultivators:   
+👪 Members: [**@business-planning-wg**][t-biz]  
+✅ Tasks: GitHub board > [`wg:business-planning` label][l-biz]  
+
+### [Finance][fin-wg]
+
+WG for establishing accounting practices, including setting up a bank account, administering payroll and expenses, invoicing, and handling remittances.
```

Comment:
```suggestion
WG for stewarding the financial sustainability of the organization, including establishment of accounting practices, maintenance of our financial accounts, facilitation of invoicing, expenses, and payroll, and reporting of our financial statements and tax obligations.
```

Reply:
Makes sense! However, there isn't agreement in this list, can I propose a friendly amendment (Allll about grammar):

```suggestion
WG for stewarding the financial sustainability of the organization, this includes establishing accounting practices, maintaining financial accounts, processing invoices, expenses, and payroll, and creating reports on our financial statements and tax obligations.
```

Reply:
Maybe after we commit these, we could toss it back to WG's to revisit and amend before some deadline? We should prob get in the habit of updating these through more of a shared process (instead of PR), and now seems like as good a time as ever :)

Reply:
Yes, the intent is not that @hyphacoop/ops-wg does this, just making space for each group to contribute

Reply:
Thanks for fixing this @dcwalk much better.